### ISSUE TITLE
nothing cached for metric cache [bash scripts/evaluation/run_metric_caching.sh]

### ISSUE THREAD

**Issue by xinfu607** [autonomousvision/navsim#118]
Hi, thanks a lot for the great work. 
Nothing has been cached after running the run_metric_caching_bk.sh. Would you pls let me know the reason. Thx a lot. 

Here is the log.
2025-07-08 06:46:57,967 INFO {/home/xin.fu2/code/model_training/planning/model/navsim/navsim/planning/metric_caching/caching.py:166}  Starting metric caching of 0 files...
Ray objects: 0it [00:00, ?it/s]
2025-07-08 06:46:57,968 INFO {/home/xin.fu2/code/model_training/planning/model/navsim/navsim/planning/metric_caching/caching.py:174}  Completed dataset caching! All 0 features and targets were cached successfully.
2025-07-08 06:46:57,968 INFO {/home/xin.fu2/code/model_training/planning/model/navsim/navsim/planning/metric_caching/caching.py:193}  Node 0: Storing metadata csv file containing cache paths for valid features and targets...
2025-07-08 06:46:58,012 INFO {/home/xin.fu2/.conda/envs/py39/lib/python3.9/site-packages/nuplan/planning/training/experiments/cache_metadata_entry.py:66}  Using cache_metadata_storage_path: /perception-hl/xin.fu2/dataset/navsim/exp_bk/metric_cache/metadata/metric_cache_metadata_node_0.csv
2025-07-08 06:46:58,074 INFO {/home/xin.fu2/code/model_training/planning/model/navsim/navsim/planning/metric_caching/caching.py:195}  Done storing metadata csv file.

run_metric_caching.sh
TRAIN_TEST_SPLIT=navhard_two_stage
CACHE_PATH=$NAVSIM_EXP_ROOT/metric_cache
python $NAVSIM_DEVKIT_ROOT/navsim/planning/script/run_metric_caching.py \
train_test_split=$TRAIN_TEST_SPLIT \
metric_cache_path=$CACHE_PATH

Best,
Xin
